deepmd/pt/model/task/population.py

```
@@ -0,0 +1,148 @@
+    mixed_types : bool
+        If true, use a uniform fitting net for all atom types, otherwise use
+        different fitting nets for different atom types.
+    seed : int, optional
+        Random seed.
+    """
+
+    def __init__(
+        self,
+        ntypes: int,
+        dim_descrpt: int,
+        neuron: list[int] = [128, 128, 128],
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)